osmaxx/conversion_api/formats.py

```diff
@@ -54,6 +75,8 @@ def detail_level_available(self):
         archive_file_name_identifier='Shapefile',
         abbreviations=[],
         is_white_box=True,
+        layer_filename_extension='.shp',
+        attribute_value_encoding='ISO-8859-1',
```

Comment:
This (a.k.a. `latin1`) seems to be the default encoding of Shapefiles if no `*.cpg` file is present and the LDID/codepage setting from the `*.dbf` file doesn't specify anything. It's probably also what OGR creates (when writing Shapefiles) when not told otherwise.

However, `latin1` will not be able to properly encode some values that will occur in the data (e.g. non-western names in respective language's native script). @sfkeller Thus, should we instead force OGR to create Shapefiles with e.g. UTF-8 encoded attribute values in the `*.dbf` files along with appropriate `*.cpg` files specifying that encoding?